tests/playwright-tests/tests/admin/themes.spec.ts

```diff
@@ -1,10 +1,10 @@
 import { test } from '@playwright/test';
-import environment from '../utils/env';
-import AzureLoginPage from '../admin/azpage/AzureLoginPage';
-import AdminPage from '../admin/pages/AdminPage';
-import ThemesPage from '../admin/pages/ThemesPage';
-import CreateThemePage from '../admin/pages/CreateThemePage';
-import uiTestString from '../utils/uiTestString';
+import environment from '../../utils/env';
```

Comment:
I expect there's probably a way that you can specify these file paths from the root rather than relative to this file, in the same way that we use imports like "@admin/component/Link" by using the "@admin" alias.

Amy B could probably help get this set up if you needed an extra pair of eyes.